CHANGELOG.md

```diff
@@ -17,6 +17,8 @@ Versioning](http://semver.org/spec/v2.0.0.html).
 - `handler::parallel_for(size_t, [size_t,] ...)` now acts as a shorthand for `parallel_for(range<1>, [id<1>,] ...)` (#288)
 - Experimental support for the AdaptiveCpp generic single-pass compiler (#294)
 - Constructor overloads to the `access::neighborhood` range mapper for reads in 3/5/7-point stencil codes (#292)
+- The SYCL backend now uses per-device submission threads to dispatch commands for better performance. 
+  This new behaviour is enabled by default, and can be disabled via "CELERITY_BACKEND_DEVICE_SUBMISSION_THREADS" (#303)
```

Comment:
```suggestion
- The SYCL backend now uses per-device submission threads to dispatch commands for better performance in multi-device configurations. 
  This new behavior is enabled by default, and can be disabled via "CELERITY_BACKEND_DEVICE_SUBMISSION_THREADS" (#303)
```

Come to think of it, should we disable the thread if there's only a single device..?

Reply:
Evaluated this on the thread pinning branch: even with just 1 device, having the BE submission thread is actually slightly faster (i.e. 600 ms vs 650 ms in the 2D stencil OS3 case), so I don't think we should complicate the defaults.